pom.xml

```diff
@@ -170,11 +175,41 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>${project.basedir}</directory>
+                <includes>
+                    <include>pom.xml</include>
+                </includes>
+                <targetPath>${project.build.directory}/resolved-pom/</targetPath>
```

Comment:
creates a pom file in the output folder (`target/`) with the all maven properties filtered (i.e.  replaced with their actual values)